sdk_commons/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
```

Comment:
do we need that for sdk_commons?

Reply:
Should we define a top level project config?

Reply:
It is basically a copy of other pyprojects and for sure can be simplified. More in general I'm not very happy with having so many packages but probably we would like to improve this part on PTP-9266, wdyt?